cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from django.http import HttpRequest
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+
+from .models import Movie
+from .serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movies(request: HttpRequest) -> Response:
```

Comment:
rename this, so it is easy to understand in other parts of application that this is a view and what it does